.vscode/settings.json

```diff
@@ -15,7 +15,7 @@
     "editor.suggestSelection": "first",
     "eslint.workingDirectories": [{"mode": "auto"}],
     "vsintellicode.modify.editor.suggestSelection": "automaticallyOverrodeDefaultValue",
-    "workbench.colorTheme": "Default Dark+",
+    "workbench.colorTheme": "Atom One Dark",
```

Comment:
Do you think you can remove this? I think you can just not add it to git when you do git add, or you can also uncheck the box on Github Desktop